homeassistant/components/bond/__init__.py

```diff
@@ -50,14 +50,16 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     if not entry.unique_id:
         hass.config_entries.async_update_entry(entry, unique_id=hub.bond_id)
 
+    hub_name = f"{hub.name} ({hub.bond_id})" if hub.name else hub.bond_id
```

Comment:
Isn't it weird to put the ID in the name ?

Reply:
Wouldn't this make more sense?
```suggestion
    hub_name = hub.name or hub.bond_id
```

Reply:
I agree we don't need it here